daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -56,9 +56,10 @@ object VersionTimeline {
       Both(This(ValueVersion("5")), LanguageVersion(LMV.V1, "6")),
       Both(This(ValueVersion("6")), LanguageVersion(LMV.V1, "7")),
       This(That(TransactionVersion("9"))),
+      That(LanguageVersion(LMV.V1, "8")),
```

Comment:
@remyhaemmerle-da I don’t really understand this file. Based on the TODO, I believe that this should go before `ValueVersion("7")` since we are not including `GenMap` in DAML-LF 1.8. But please check that this matches the intent.

Reply:
Please delete the `FIXME` with the `This(This(ValueVersion("7"))),`

Reply:
Do you want me to delete just the FIXME or also the `This(This(ValueVersion("7"))` line?

Reply:
Yep

Reply:
You did good.